proxy/tests/graphql_query.rs

```diff
@@ -527,6 +526,34 @@ fn tree_root() {
     });
 }
 
+#[test]
+fn list_transactions() {
+    with_fixtures(|librad_paths, _repos_dir, _platinum_id| {
+        let mut vars = Variables::new();
+        vars.insert("ids".into(), InputValue::list(vec![]));
+        let query = "query($ids: [ID!]!) {
+            listTransactions(ids: $ids) {
+                messages {
+                    ... on ProjectRegistration {
+                        projectName,
+                        orgId
+                    }
+                },
```

Comment:
It is a bit confusing to request a specific content although we are asserting that there will be no transactions

Reply:
Still working on meaningful tests, this is wip.